index.html

```diff
@@ -1,13 +1,383 @@
+      <div class="signin-form">
+        <div class="img-box"></div>
+        <h1><a href="">1,492 smashing members</a></h1>
+        <p>
+          A friendly community that keeps this website alive. With books,
+          webinars, discounts and networking — for just 1 coffee a month. Join
+          the family.
+        </p>
+        <button href="" role="button" class="btn">
+          Explore Membership&nbsp;→
+        </button>
+      </div>
+    </section>
+    <section class="later-posts">
+      <div class="later-post later-post-1">
+        <h2><a href="">Image Optimization In WordPress</a></h2>
+        <p>
+          In this article, Adelina Țucă explains how you can easily optimize all
+          the images on your website (manually or on autopilot) in order to gain
+          better loading times.
+        </p>
+      </div>
+      <div class="later-post later-post-2">
+        <h2><a href="">How To Start Using Sketch And Framer X</a></h2>
+        <p>
+          When it comes to showing the transition, interaction and animation of
+          elements in the user interface, a prototyping tool like Framer X can
+          make a difference in the way you communicate your vision to the team
+          and stakeholders and as a result, boost …
+        </p>
+      </div>
+      <div class="later-post later-post-3">
+        <h2><a href="">Increase Customer Centricity With Workshops</a></h2>
+        <p>
+          In this article, Claire Mason reviews four types of workshops that
+          drive customer centricity at the general and project-specific levels.
+        </p>
+      </div>
+      <div class="later-post later-post-4">
+        <h2>
+          <a href=""
+            >When Do We Need A Design System? An Interview With Brad Frost</a
+          >
+        </h2>
+        <p>
+          The web is wonderfully diverse and unpredictable because of
+          wonderfully diverse people shaping it. In this new series of short
+          interviews, we talk to interesting people doing interesting work in
+          our industry and sharing what they’ve learned.
+        </p>
+      </div>
+      <div class="later-post later-post-5 smash-tv">
+        <div class="img-box"></div>
+        <h1><a href="">New on Smashing TV</a></h1>
+        <p>
+          A channel where seasoned developers show how they design, build, work
+          and play. Coming up next: Video Editing With CSS And JavaScript with
+          Eva Ferreira (Jun 19).
+        </p>
+        <button href="" role="button" class="btn">
+          See all upcoming episodes&nbsp;→
+        </button>
+      </div>
+    </section>
+    <section class="conference">
+      <h1>SmashingConf Toronto 2019</h1>
+      <p>
+        Improving accessibility and front-end workflow, refactoring, security,
+        interface design patterns, CSS Grid techniques, performance, Git and how
+        to write better JavaScript
+      </p>
+      <button href="" role="button" class="btn">
```

Comment:
Note that `href` is not allowed on element `button`, kindly do it in the right way.